rubix-core/src/main/java/com/qubole/rubix/core/CachingInputStream.java

```diff
@@ -99,7 +99,7 @@ public CachingInputStream(FileSystem parentFs, Path backendPath, Configuration c
       this.lastModified = fileInfo.lastModified;
     }
     catch (Exception ex) {
-      log.error("Could not get FileInfo for " + backendPath.toString() + " . Fetching FileStatus from remote file system");
+      log.error("Could not get FileInfo for " + backendPath.toString() + " . Fetching FileStatus from remote file system \n StackTrace : ", ex);
```

Comment:
Having the exception as the second argument of the `log.error` call should automatically put the stack trace on a new line.
Also, I feel like we shouldn't need to specify that it is a stack trace; it should be clear enough from what is logged.